src/indent.cpp

```diff
@@ -1560,56 +1560,51 @@ void indent_text(void)
          frm.pse[frm.pse_tos].indent_tmp = frm.pse[frm.pse_tos].indent;
          frm.pse[frm.pse_tos].indent_tab = frm.pse[frm.pse_tos].indent;
          log_indent_tmp();
-
          indent_column_set(frm.pse[frm.pse_tos].indent_tmp);
 
-         if ((cpd.settings[UO_indent_class_colon].b && (pc->type == CT_CLASS_COLON)) ||
-             (cpd.settings[UO_indent_constr_colon].b && (pc->type == CT_CONSTR_COLON)))
+         prev = chunk_get_prev(pc);
+         next = chunk_get_next(pc);
```

Comment:
Could you please expand on that?

`prev` is used only at line 1583 in `chunk_is_newline(prev)` which is safe, because it checks for null pointer inside. I could add an explicit check here also if you'd prefer so, but seems pointless.

`next` is used two times below, but both times after `if ((next != nullptr) && ...)` already.